neurom/core/_neuron.py

```diff
@@ -162,9 +163,10 @@ def __init__(self, root_node):
     def points(self):
         '''Return unordered array with all the points in this neurite'''
         # add all points in a section except the first one, which is a duplicate
-        _pts = [v for s in self.root_node.ipreorder() for v in s.points[1:, :4]]
+        _pts = [v for s in self.root_node.ipreorder()
+                for v in s.points[1:, COLS.XYZR]]
```

Comment:
Yeah, the language isn't as tight as I would like, but referencing points often includes the radius, so it's included.